src/MSBuildLocator/MSBuildLocator.cs

```diff
@@ -305,7 +308,7 @@ public static void Unregister()
         /// <summary>
         ///     Ensures the proper MSBuild environment variables are populated for DotNet SDK.
         /// </summary>
-        /// <param name="msbuildPath">
+        /// <param name="dotNetSdkPath">
         ///     Path to the directory containing the DotNet SDK.
```

Comment:
We should probably be brand consistent.

```suggestion
        ///     Path to the directory containing the .NET SDK.
```